portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,126 @@
+<div class="w3-row">
+
+<!-- Blog entries -->
+<div class="w3-col l8 s12">
+  <!-- Blog entry -->
+  <div class="w3-card-4 w3-margin w3-white">
+        <img src="/images/unnamed.png" alt="Logo" width="900">
+            <div class="w3-container">
+      <h3><b>Cheeselover012</b></h3>
+      <h5>Learn how Cheeselover012 came to be!<span class="w3-opacity">Circa 2012</span></h5>
+    </div>
+
+    <div class="w3-container">
+      <p>Cheeselover012 started when a girl decided she wanted a youtube channel. She loved Cheese and decided that should
+          be her username. However, that was taken, so she added a 0, which was still taken. Her favorite number is 12, so 
+          she decided to add that to the username. This is how Cheeselover012 came to be.</p>
+      <div class="w3-row">
+        <div class="w3-col m8 s12">
+        </div>
+        <div class="w3-col m4 w3-hide-small">
+        </div>
+      </div>
+    </div>
+  </div>
+  <hr>
+
+  <!-- Blog entry -->
+  <div class="w3-card-4 w3-margin w3-white">
+    <div class="w3-container">
+      <h3><b>Serena and Congress</b></h3>
+      <h5>Haven't you heard? Serena Aeschilman wants to run for Congress in 2024. Consider voting for her if you are in her
+          state of Utah. <span class="w3-opacity">VOTE IN 2024</span></h5>
+    </div>
+
+    <div class="w3-container">
+      <p>Serena believes that the government should be getting more involved in the technology sector. What should the government
+          be regulating? Well our data for one. How much data are these companies getting from a click from an unsuspecting user?
+          Are websites ADA approved? What are the effects of some of the products on the US economy? Ask Serena directly if you 
+          are curious and want to learn more about why she is running for Congress. 
+      </p>
+      <div class="w3-row">
+        <div class="w3-col m8 s12">
+        </div>
+        <div class="w3-col m4 w3-hide-small">
+        </div>
+      </div>
+    </div>
+  </div>
+<!-- END BLOG ENTRIES -->
+</div>
+
+<!-- Introduction menu -->
+<div class="w3-col l4">
+  <!-- About Card -->
+  <div class="w3-card w3-margin w3-margin-top">
+    <img src="/images/Serena.jpeg" alt="Serena">
+        <div class="w3-container w3-white">
+      <h4><b>Serena Aeschilman</b></h4>
+      <p>My name is Serena Aeschilman. I am currently a 3rd year at the University of Utah. I am studying Computer Science
+          and Economics. I am hoping to utilize these two skills in Congress. I am running for Congress in 2024, so if you
+          are in the area, consider voting for me! Thanks for checking out this portfolio!</p>
+    </div>
+  </div><hr>
+  
+  <!-- Posts -->
+  <div class="w3-card w3-margin">
+    <div class="w3-container w3-padding">
+      <h4>Facts</h4>
+    </div>
+      <p>Click here to get a random fact about Serena Aeschilman:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
```

Comment:
Would it make sense to rename the JS method and this container since they no longer have to do with greetings

Reply:
done